detect/synthetic-monitoring/playwright-checks/custom-dependencies.mdx

```diff
@@ -10,18 +10,20 @@ Use the [Checkly CLI](/cli/) to turn your Playwright tests into globally distrib
 
 Checkly installs dependencies from your `package.json` and lock files. It works with [npm](https://www.npmjs.com/), [yarn](https://yarnpkg.com/), and [pnpm](https://pnpm.io/).
 
-```json
+By default, Checkly installs all dependencies (both `dependencies` and `devDependencies`). You can change this account-wide setting to install only `devDependencies` in your [account settings](https://app.checklyhq.com/accounts/settings/account/playwright).
```

Comment:
```suggestion
By default, Checkly installs all dependencies (both `dependencies` and `devDependencies`). You can change this setting to install only `devDependencies` in your [account settings](https://app.checklyhq.com/accounts/settings/account/playwright).
```

Since it's in account settings that felt like repetition?